src/views/Home/components/FAQ/index.tsx

```diff
@@ -1,9 +1,85 @@
 import * as React from "react"
 import "./index.scss"
+import FAQInformation from "Props/faq/faq"
+import { FAQProps } from "Props/props"
 
+const col1 = FAQInformation.slice(0, 4)
+const col2 = FAQInformation.slice(4, 8)
+const col3 = FAQInformation.slice(8, 12)
```

Comment:
This should be (4,9) and (9,13). This is why I think we are missing a question in the render.